app/src/components/layout/Navbar/Navbar.tsx

```diff
@@ -25,30 +26,32 @@ import { ConfigurableLogo, DefaultNavLogo } from '../LogoComponents';
 
 const navItems = [
   {
-    title: 'home',
+    title: 'Home',
     path: '/',
     key: 'home',
   },
   {
-    title: 'blocks',
+    title: 'Blocks',
     path: '/blocks',
     key: 'blocks',
   },
   {
-    title: 'peers',
+    title: 'Peers',
     path: '/peers',
     key: 'peers',
   },
   {
-    title: 'validators',
+    title: 'Validators',
     path: '/validators',
     key: 'validators',
   },
 ];
 
 export const Navbar: React.FC = () => {
+  const [isOpened, setIsOpened] = useState<boolean>();
```

Comment:
The typing would infer `undefined | boolean`, but we only want a boolean

```suggestion
  const [isOpened, setIsOpened] = useState(false);
```

Reply:
Reverted!